etc/migration/disable_users_from_tenancy/lib/disable-users-by-tenancy.js

```diff
@@ -0,0 +1,132 @@
+/*!
+* Copyright 2017 Apereo Foundation (AF) Licensed under the
+* Educational Community License, Version 2.0 (the "License"); you may
+* not use this file except in compliance with the License. You may
+* obtain a copy of the License at
+*
+*     http://opensource.org/licenses/ECL-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS"
+* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+* or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+/*
+* Disable users belonging to a disabled tenancy
+* Github issue #1304
+*/
+
+var _ = require('underscore');
+var bunyan = require('bunyan');
+var async = require('async');
+var Cassandra = require('oae-util/lib/cassandra');
+var log = require('oae-logger').logger('oae-script-main');
+var PrincipalsDAO = require('oae-principals/lib/internal/dao');
+
+/**
+ * Disable users from the system by updating the deleted flag
+ * 
+ * @function doMigration
+ * @param  {type} tenantAlias {description}
+ * @param  {type} callback    {description}
+ * @return {type} {description}
+ */
+var doMigration = function (tenantAlias, callback) {
+    _getAllUsersForTenant(tenantAlias, function(error, users) {
+        if (error) {
+            log().error({
+                'err': err
```

Comment:
We're using `error` above, not `err` - I'd actually change it to `err` across board for consistency.